sns/types.bal

```diff
@@ -14,6 +14,18 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerinax/'client.config;
+
+# Represents the AWS SNS Connection Configuration.
+@display {label: "Connection Config"}
+public type ConnectionConfig record {|
+    *config:ConnectionConfig;
+    # Credentials to authenticate client 
+    AwsCredentials|AwsTemporaryCredentials credentials;
```

Comment:
```suggestion
    @display{label: "", kind: "password"}
    AwsCredentials|AwsTemporaryCredentials credentials;
```

Reply:
Since this is a record type field, I have marked the relevant field's display kind in the inner record